packages/amplication-client/src/VersionControl/PendingChanges.tsx

```diff
@@ -24,6 +25,7 @@ import useBreadcrumbs from "../Layout/use-breadcrumbs";
 
 const CLASS_NAME = "pending-changes";
 const POLL_INTERVAL = 2000;
+const STATE_COMMIT_DONE = "done";
```

Comment:
```suggestion
const COMMIT_STATE_DONE = "done";
```

Reply:
Can you use a value from an enum from the server?

Reply:
it has nothing to do with the server, It is literally a "client side" state that only describe the state of the page in order to decide what image to show when the list is empty